mbs_results/growth_rate_output.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+import pandas as pd
+
+
+def get_growth_rate_data(filepath):
+    """
+    Filters and pivots wider winsorisation data on period to return growth rate data.
+
+    Parameters
+    ----------
+    filepath : str
+        filepath to the winsorisation output.
+
+    Returns
+    -------
+    pandas.Data.Frame
+        Dataframe containing classification, question number, cell number, and pivoted
+        wider on period with adjusted values.
+    """
+
+    input_data = pd.read_csv(
+        filepath,
+        usecols=[
+            "classification",
+            "question_no",
```

Comment:
Is it worth passing through question_no as an input in case the column changes in the input data? Trying to remove any possible hard coding columns, (same for period) but as this is an output it might not be needed (Optional)

Reply:
Discussed this in Stand-up, can return and consider changing hard coded col names further down the line